### ISSUE TITLE
Start-up not showing previously installed services

### ISSUE THREAD

**Issue by AdyGCode** [TablePlus/DBngin#57]
Please fill out the detail below, it helps me investigate the bug:

1. Driver (Ex: PostgreSQL 10.0):
- MariaDB and others

2. DBngin build number:
- v3.1 (32)

3. macOS version:
- 11.1 Big Sur

4. The steps to reproduce this issue:
- Install DBngin for 1st time
- Run dBngin
- Blank list of services, and a message identifying to click +
- Must click Refresh button to update the list of services

Suggested solution:
- On (initial) run, have DBngin check for any installed services and list these installed services

**Comment by seandelaney** [comment]
Agreed. I have Postgres service running and first time installing DBngin and running the app, service list is empty. I also expected to see both Postgres and MySQL services which are currently running listed.
